content/src/content/jcr_root/apps/core/wcm/components/form/text/v2/text/amp.html

```diff
@@ -0,0 +1,49 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div class="cmp-form-text"
+     data-sly-use.text="com.adobe.cq.wcm.core.components.models.form.Text"
+     data-cmp-is="formText">
+    <label for="${text.id}" data-sly-test="${text.title && !text.hideTitle}">${text.title}</label>
+    <p data-sly-test="${!text.placeholder && text.helpMessage}" class="cmp-form-text__help-block">${text.helpMessage}</p>
+    <textarea data-sly-test.useTextarea="${text.type == 'textarea'}"
+        class="cmp-form-text__textarea"
+        data-cmp-hook-form-text="input"
+        aria-label="${text.hideTitle && text.title}"
+        id="${text.id}"
+        placeholder="${text.placeholder}"
+        name="${text.name}"
+        readonly="${text.readOnly}"
+        required="${text.required}"
+        rows="${text.rows}">${text.value}</textarea>
+    <input data-sly-test="${!useTextarea}"
+        class="cmp-form-text__text"
+        data-cmp-hook-form-text="input"
+        aria-label="${text.hideTitle && text.title}"
+        required="${text.required}"
+        type="${text.type}"
+        id="${text.id}"
+        placeholder="${text.placeholder}"
+        name="${text.name}"
+        value="${text.value}"
+        readonly="${text.readOnly}"/>
+
+    <span visible-when-invalid="valueMissing"
+          validation-for="${text.id}">${text.requiredMessage}</span>
```

Comment:
If the message is not configured in authoring, AMP falls back to it's own validation messaging.